### PR TITLE
Feature/21556 Remove pupil census

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- Remove pupil census feature migrations and service changes

### CODE REVIEW

File: admin/controllers/service-manager.js [hunk 1]
```diff
@@ -172,6 +172,24 @@ const controller = {
...
+   * @returns {Promise<*>}
+   */
+  getRemovePupilCensus: async (req, res, next) => {
+    const pupilCensusId = req.params && req.params.pupilCensusId
+    try {
+      await pupilCensusService.remove(pupilCensusId)
+    } catch (error) {
+      return next(error)
+    }
+    req.flash('info', `Pupil data successfully removed`)
```
Comment (jon-shipley) [inline admin/controllers/service-manager.js]: No need for backticks here - plain quotes are fine.

File: admin/services/pupil-census.service.js [hunk 2]
```diff
@@ -97,11 +98,35 @@ pupilCensusService.getUploadedFile = async () => {
...
+  if (jobStatus.jobStatusCode === completedWithErrors) {
+    outcome = pupilCensus.errorOutput
+  } else if (jobStatus.jobStatusCode === deleted) {
+    outcome = jobStatus.description
+  } else {
+    outcome = `${jobStatus.description} : ${pupilCensus.jobOutput}`
+  }
   pupilCensus.csvName = pupilCensus.jobInput
   pupilCensus.outcome = outcome
+  pupilCensus.jobStatusCode = jobStatus.jobStatusCode
```
Comment (jon-shipley) [inline admin/services/pupil-census.service.js]: Is there any chance that `jobStatusCode` is tainted?  Or is it **guaranteed** to be a correct value?

Comment (GeorgeChatzigiannis) [reply admin/services/pupil-census.service.js]: It's been populated directly from the db record so I believe it's guaranteed.

Comment (jon-shipley) [reply admin/services/pupil-census.service.js]: that's fine then

**Pull request merged**